cmd/kube-controller-manager/controller-manager.go

```diff
@@ -51,8 +50,9 @@ var (
 	minionRegexp    = flag.String("minion_regexp", "", "If non empty, and -cloud_provider is specified, a regular expression for matching minion VMs.")
 	machineList     util.StringList
 	// TODO: Discover these by pinging the host machines, and rip out these flags.
+	// TODO: in the meantime, use resource.QuantityFlag() instead of these
 	nodeMilliCPU = flag.Int64("node_milli_cpu", 1000, "The amount of MilliCPU provisioned on each node")
-	nodeMemory   = flag.Int64("node_memory", 3*1024*1024*1024, "The amount of memory (in bytes) provisioned on each node")
+	nodeMemory   = resource.QuantityFlag("node_memory", "3Gi", "The amount of memory (in bytes) provisioned on each node")
```

Comment:
QuantityFlag has to return a pointer, because it's going to modify the variable it returns when flag.Parse() is called.

I did it this way because it's just like the pattern that the flag package itself uses.